README.md

```diff
@@ -356,6 +356,96 @@ countItems := response.CountItems
 count := response.Count
 ```
 
+## Importing and Exporting Data
+
+### Importing Data
+
+If you have large amounts of data, it is more efficient to import it to Pilosa instead of several `SetBit` queries. This library supports importing bits of a frame. The target frame must have been created before hand.
+
+Before starting the import, create an instance of a struct which implements `BitIterator` and pass it to `client.ImportFrame` function. This library ships with `CSVBitIterator` struct which supports importing bits in the CSV (comma separated values) format:
+```
+ROW_ID,COLUMN_ID
+```
+
+Optionally, a timestamp with GMT time zone can be added:
+```
+ROW_ID,COLUMN_ID,TIMESTAMP
+```
+
+Note that, each line corresponds to a single bit and the lines end with a new line (`\n` or `\r\n`).
+
+Here's some sample code:
+```go
+text := `10,7
+    10,5
+    2,3
+    7,1`
+iterator := NewCSVBitIterator(strings.NewReader(text))
+```
+
+After creating the iterator you can pass it to `client.ImportFrame` together with the frame and batch size. The following sample sends batches with size 10000 bits:
+```go
+err = client.ImportFrame(frame, iterator, 10000)
+if err != nil {
+    panic(err)
+}
+```
+
+You can define a custom `BitIterator` by including a function with the signature `NextBit() (Bit, error)` in your struct.
+```go
+type StaticBitIterator struct {
+    NextBit() (Bit, error)
+    bits []Bit
+    index int
+}
+
+func NewStaticBitIterator() *StaticBitIterator {
+    return &StaticBitIterator{
+        bits: []Bit{
+            Bit{RowID: 1, ColumnID: 1, Timestamp: 683793200},
+            Bit{RowID: 5, ColumnID: 20, Timestamp: 683793300},
+            Bit{RowID: 3, ColumnID: 41, Timestamp: 683793385},
+        }
+    }
+}
+
+func (it *StatiBitIterator) NextBit(Bit, error) {
+    if it.index < len(it.bits) {
+        return Bit{}, io.EOF
+    }
+    return it.bits[it.index++], nil
+}
+```
+
+### Exporting Data
+
+You can export a view of a frame from Pilosa using `client.ExportFrame` function which returns a `BitIterator`. use `NextBit` function of this iterator to receive all bits for the specified frame. When there are no more bits, `io.EOF` is returned.
```

Comment:
You can export a view of a frame from Pilosa using the `client.ExportFrame` function which returns a `BitIterator`. Use the `NextBit` function of this iterator to receive all bits for the specified frame. When there are no more bits, `io.EOF` is returned.